eng/pipelines/aggregate-report.yml

```diff
@@ -0,0 +1,51 @@
+trigger: none
+pr: none
+
+jobs:
+  - job: Generate
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'windows-2019'
+    steps:
+      - pwsh: mkdir $(System.ArtifactsDirectory)/BuildArtifacts
+        displayName: Create Directory
+
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download Pipeline Artifact'
+        inputs:
+          buildType: specific
+          project: $(TargetPipelineProjectId)
+          definition: $(TargetPipelineDefinition)
+          buildVersionToDownload: latestFromBranch
+          tags: scheduled
+          artifactName: packages
+          targetPath: '$(System.ArtifactsDirectory)/BuildArtifacts'
+
+      - pwsh: Remove-Item Microsoft.Azure.*
+        workingDirectory: $(System.ArtifactsDirectory)/BuildArtifacts
+        displayName: Remove Track 1 Packages
+
+      - pwsh: |
+          mkdir "$(Build.ArtifactStagingDirectory)/reports"
+          Copy-Item -Path "$(Build.SourcesDirectory)/eng/common/InterdependencyGraph.html" -Destination "$(Build.ArtifactStagingDirectory)/reports/InterdependencyGraph.html"
+          & "tools/AnalyzeDeps/AnalyzeDeps.ps1" -PackagesPath "$(System.ArtifactsDirectory)/BuildArtifacts/*.nupkg" -LockfilePath "$(Build.SourcesDirectory)/eng/Packages.Data.props" `
+          -OutPath "$(Build.ArtifactStagingDirectory)/reports/dependencies.html" -DumpPath "$(Build.ArtifactStagingDirectory)/reports/data.js"
+        displayName: "Analyze dependencies"
+        condition: succeeded()
```

Comment:
`succeeded()` is the default condition, so these instances here and below aren't necessarily needed.

Reply:
Oh, I see thanks